includes/admin/reporting/reports.php

```diff
@@ -2529,6 +2527,7 @@ function edd_register_export_report( $reports ) {
 		    'priority'         => 1000,
 		    'capability'       => 'export_shop_reports',
 		    'display_callback' => 'display_export_report',
+			'filters'          => false,
```

Comment:
Some wonky spacing here. Looks like it's because the previous arguments use spaces in a place they shouldn't be. Can you adjust that?